tools/frrcommon.sh.in

```diff
@@ -66,12 +66,16 @@ chownfrr() {
 
 vtysh_b () {
 	[ "$1" = "watchfrr" ] && return 0
-	[ -r "$C_PATH/frr.conf" ] || return 0
-	if [ -n "$1" ]; then
-		"$VTYSH" `echo $nsopt` -b -d "$1"
-	else
-		"$VTYSH" `echo $nsopt` -b
+	if [ -r "$C_PATH/frr.conf" ]; then
+		log_warning_msg "$C_PATH/frr.conf does not exist; skipping config apply"
+		return 0
 	fi
+
+	cmd="$VTYSH $nsopt -b"
+	[ -n "$1" ] && cmd+=" -d $1"
```

Comment:
`+=` is a bashism, it doesn't exist in POSIX sh. This needs to be `cmd="${cmd} -d $1"` instead.